src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -146,10 +145,12 @@ $(document).ready(() => {
         adminSearchCollapseAllStudents();
     });
 
-    $('.openEmailApplicationDefaultValues').submit((e) => {
+    $('.open-email-application-default-values').submit((e) => {
         e.preventDefault();
         const $entry = $(e.target);
+        const crlf = encodeURIComponent('\r\n');
```

Comment:
I'm not sure why you didn't address my question on why we must use CRLF but I think we have to address this otherwise this looks Windows specific/magical.

As per [RFC 6068](https://tools.ietf.org/html/rfc6068), an excerpt
> Also note that line breaks in the body of a message MUST be encoded with "%0D%0A".

Reply:
> Is it because browsers always encode newlines as %0D%0A regardless of platform?

@LiHaoTan Yes, sorry for not replying earlier.

Reply:
No worries :)